wagtailio/core/blocks.py

```diff
@@ -374,3 +374,165 @@ class IconBulletsBlock(blocks.StructBlock):
     class Meta:
         # template = "" # TODO: add template
         label = "Icon bullets"
+
+
+class StandaloneQuoteBlock(blocks.StructBlock):
+    quote = blocks.TextBlock(required=True)
+    author = blocks.RichTextBlock(
+        required=True,
+        features=[
+            "bold",
+            "italic",
+            "h2",
+            "h3",
+            "h4",
+            "h5",
+            "ol",
+            "ul",
+            "link",
+            "image",
+            "document",
+            "embed",
+        ],
+    )
+    author_image = ImageChooserBlock(required=False)
+
+    class Meta:
+        icon = "openquote"
+        label = "Standalone quote"
+        template = "patterns/components/streamfield/standalone_quote_block.html"
+
+
+class MultipleQuoteBlock(blocks.StructBlock):
+    heading = blocks.TextBlock(required=True)
+    quotes = blocks.ListBlock(StandaloneQuoteBlock(), min_num=2)
+
+    class Meta:
+        icon = "openquote"
+        label = "Multiple quotes"
+        template = "patterns/components/streamfield/multiple_quotes_block.html"
+
+
+class RichTextBlock(blocks.StructBlock):
+    rich_text = blocks.RichTextBlock(
+        required=True,
+        features=["bold", "italic", "h2", "h3", "ol", "ul", "link", "document"],
+    )
+
+    class Meta:
+        icon = "bold"
+        label = "Rich text"
+        template = "patterns/components/streamfield/rich_text_block.html"
+
+class StandaloneCTABlock(blocks.StructBlock):
+    short_description = blocks.TextBlock(required=False, max_length=100)
+    cta = CTABlock()
+
+    class Meta:
+        icon = "arrow-right"
+        label = "Standalone CTA"
+        template = "patterns/components/streamfield/standalone_cta_block.html"
+
+
+class TeaserBlock(blocks.StructBlock):
+    blog_post_chooser = blocks.PageChooserBlock(required=False, page_type=["blog.BlogPage"])
```

Comment:
```suggestion
    page = blocks.PageChooserBlock(required=False, page_type=["blog.BlogPage"])
```

should this expand in scope later on, `blog_post_chooser` would no longer make sense

Reply:
updated in f3751fde600de09043686218a674a322b0039046